app/models/record.rb

```diff
@@ -30,7 +30,15 @@ class Record < ApplicationRecord
   scope :ordered_by_created_at, -> { order('records.created_at DESC') }
   scope :active_ordered, -> { active.ordered_by_created_at }
   scope :top_five, -> { limit(5) }
-  scope :with_associations, -> { eager_load(:user, :ramen_shop).preload(:line_statuses, image_attachment: :blob) }
+  scope :with_associations, lambda {
+    eager_load(:user, :ramen_shop).preload(:line_statuses, :likes, image_attachment: :blob)
+  }
+  scope :with_most_likes, lambda {
```

Comment:
[nits and ask]

全然問題ないですがここだけlambdaを省略しなかったのには理由がありますか？？

```suggestion
  scope :with_most_likes, -> {
```

Reply:
ありがとうございます！
RuboCopより,`Style/Lambda: Use the lambda method for multiline lambdas`と指摘を受け、'lambda'を使うことになりました。RuboCop設定を調整することは可能です！